frontend/src/metabase/components/UserPicker/UserPicker.jsx

```diff
@@ -78,8 +82,8 @@ const UserPicker = ({ value, validateValue, users, canAddItems, onChange }) => {
   );
 };
 
-const includesIgnoreCase = (s1, s2) => {
-  return s1.toLowerCase().includes(s2.toLowerCase());
+const includesIgnoreCase = (sourceText, searchText) => {
+  return sourceText.toLowerCase().includes(searchText.toLowerCase());
```

Comment:
Make the function similar to aother place and improve readability.

https://github.com/metabase/metabase/blob/ef9a0acc08640144a76c68769415d81e532d5d5e/frontend/src/metabase/admin/databases/components/widgets/EngineWidget/EngineWidget.tsx#L297-L302